data/src/main/java/in/koreatech/koin/data/repository/ArticleRepositoryImpl.kt

```diff
@@ -57,6 +51,15 @@ class ArticleRepositoryImpl @Inject constructor(
         initialValue = emptyList()
     )
 
+    private val hotArticleHeaders: StateFlow<List<ArticleHeader>> = flow {
+            emit(articleRemoteDataSource.fetchHotArticles().map { it.toArticleHeader() })
+        }.stateIn(
+            scope = coroutineScope,
+            started = SharingStarted.WhileSubscribed(5_000),
+            initialValue = listOf()
+        )
+
+
```

Comment:
테스트 되었던 부분일까요? 예외 처리가 안되어있네용! 
예외 처리 부탁합니다 🙇🏻!!
```suggestion
    private val hotArticleHeaders: StateFlow<List<ArticleHeader>> = flow {
            emit(articleRemoteDataSource.fetchHotArticles().map { it.toArticleHeader() })
        }.catch {
            // 예외 처리해주기!
        }.stateIn(
            scope = coroutineScope,
            started = SharingStarted.WhileSubscribed(5_000),
            initialValue = listOf()
        )


```

Reply:
구독하는 각 뷰모델에서 catch하도록 만드려했는데, repository에서 빈 목록을 emit하는게 낫겠네용 !
반영해두겠습니닷